lib/browser/rpc-server.js

```diff
@@ -146,6 +146,28 @@ const throwRPCError = function (message) {
   throw error
 }
 
+const rendererMissingErrorMessage = (meta, args, callIntoRenderer) => {
```

Comment:
Since this method now removes the listeners as well, I think perhaps it should be renamed and do the `console.warn` directly in here.

Maybe a name, like `logMissingRendererMessageAndRemoveListeners`